WickedEngine/wiRenderer.cpp

```diff
@@ -6998,7 +6998,7 @@ void RefreshEnvProbes(const Visibility& vis, CommandList cmd)
 				),
 				RenderPassImage::RenderTarget(
 					&vis.scene->envmapArray,
-					RenderPassImage::LoadOp::DONTCARE,
+					RenderPassImage::LoadOp::CLEAR,
```

Comment:
We only want to clear if sky is not rendered, otherwise it is more optimal to not clear.